model/Core/Classes/NamespaceMap.md

```diff
@@ -14,40 +14,24 @@ of ElementIDs to be used in SPDX content serialization in order to provide a mor
-consistency independent of any particular serialization form.
-
-A given serialization payload (whether file or streaming) MUST NOT contain multiple namespace maps with conflicting mappings.
-Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content unless the single 
-serialization payload is intended to contain multiple peer collections each with their own namespace map.
-In this case the serializer MUST do one of the following:
-1) Diverge from the exact provided namespace mappings and utilize deconflicted mappings where appropriate.
-   This could be fully new prefix mappings as if the suggested namespace maps had not been provided.
-3) Use full ElementID IRIs instead of prefixed ElementID IRIs.
-
+If a serialization format supports prefixes or namespaces (e.g., JSON-LD context or XML XSD namespace), the namespace map
+must be represented in that format "native" to the serialization.
+If the serialization format does not support prefixes, then the full URI's for the elements must be used and the namespace map will not be preserved.
+Any custom serialization format SHOULD implement namespaces in order to preserve the namespace map.
 
 Namespace maps support a variety of relevant use cases such as:
 
-1) An SPDX content producer wishing to provide clarity of their serialization of an SPDX 2.X simple style collection where all content is newly minted and a single prefix-namespace is used.
-2) An SPDX content producer wishing to maintain consistent prefix use and understanding across multiple different serializations of the produced content.
-   For example, an SBOM producer wishes to share/publish the SBOM as json-ld, XML and YAML. The producer can specify the preferred prefix mappings they plan to use in all serializations, maintain usage consistency themselves across those serializations and provide clarity to the consumer of the SBOM in any of those serializations independent of which serialization form was received.
+1) An SPDX content producer wishing to provide clarity of their serialization of an SPDX 2.X simple style collection where all content is newly minted and a single prefix-namespace is used.  The consumer of SPDX content wishes to preserve the name space mapping provided by such a producer.  In this case, the consumer would record the namespace map prefixes in the NamespaceMap such that subsequent serializations could reproduce the prefixes / namespaces in the native serialization format.
+2) An SPDX content producer wishing to maintain consistent prefix use and understanding across multiple different serialization formats of the produced content.
+   For example, an SBOM producer wishes to share/publish the SBOM as JSON-LD and XML. The producer can specify the preferred prefix mappings in the native serialization format using the information from a single Namespace stored in memory.
```

Comment:
> The producer can specify the preferred prefix mappings in the native serialization format using the information from a single Namespace stored in memory.
I do not understand what this added sentence is trying to convey.

Reply:
Change to:

The producer can specify the preferred prefix mappings in the native serialization format using information from a single Namespacemap accessible local to the producer.

Reply:
Text updated